source/renderer/app/components/staking/stake-pools/StakePoolsSearch.tsx

```diff
@@ -57,15 +57,20 @@ function StakePoolsSearchComponent({
     null
   );
 
-  const autoSelectOnFocus = () =>
+  const [focus, setFocus] = useState(false);
```

Comment:
```suggestion
  const [searchIconFocussed, setSearchIconFocussed] = useState(false);
```

Reply:
I'm not sure this should be specific to the search icon. It can be used to style other things. So rather we should name it `searchInputFocus`.

Reply:
Agree :+1: